Dockerfile

```diff
@@ -1,17 +1,20 @@
 #
-# Copyright (c) 2019-present Sonatype, Inc. All rights reserved.
-# Includes the third-party code listed at http://links.sonatype.com/products/clm/attributions.
-# "Sonatype" is a trademark of Sonatype, Inc.
+# Copyright (c) 2020-present Sonatype, Inc. All rights reserved.
+#
+# This program is licensed to you under the Apache License Version 2.0,
+# and you may not use this file except in compliance with the Apache License Version 2.0.
+# You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the Apache License Version 2.0 is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
 #
 
-# FROM docker-all.repo.sonatype.com/alpine:latest
-# LABEL maintainer="operations-group@sontype.com"
-
-# RUN apk update
+FROM docker-all.repo.sonatype.com/alpine/helm:3.9.3
 
-# WORKDIR /app
-# COPY ./src ./
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh
 
-# EXPOSE 8080
+RUN mkdir /.local /.cache && chmod 777 /.local /.cache
```

Comment:
This is what was in the existing repo for the old helm chart.

https://github.com/sonatype/helm3-charts/blob/b49eb3cc2881ee0fc6965b5e6f4059df598c95de/Dockerfile#L14-L20

Reply:
I can look into the history of these directories and the permissions they have in the original repo.

Reply:
I must admit, that something needed those directories, and they needed to write to it, so that got it to work.  The better permanent solution is probably to properly establish the `jenkins` user and home dir, and then those folders will probably be created there instead by the right user. https://github.com/sonatype/cdi-node/blob/release-17/Dockerfile

Reply:
I think I'll leave as is for now just to be able to get this published today.

Reply:
I think that's fine. This is some open permissions that live only for the time of the build to facilitate some tools.

Reply:
Cool.